src/components/app/data/utils.js

```diff
@@ -489,13 +489,15 @@ export function getCatalogsForSubsidyRequests({
     return catalogs;
   }
   if (browseAndRequestConfiguration.subsidyType === SUBSIDY_TYPE.LICENSE && customerAgreement) {
+    // availableSubscriptionCatalogs are returned from the API as a Set
```

Comment:
tiny nit: well, the API still returns a normal list/array, but is converted from a `set` to a list. Maybe update to simply mention something along the lines of "availableSubscriptionCatalogs only contains the unique catalogs across all subscription plans for an enterprise customer"?